src/store/actions/cohorts.actions.js

```diff
@@ -13,3 +13,22 @@ export const fetchCohortsThunk = () => dispatch => {
     .get(API_URL)
     .then(res => dispatch(setCohortsData(res.data.data)));
 };
+
+export const fetchActiveCohort = cohortID => dispatch => {
+  //TODO: add error handling
+  dispatch({ type: SINGLE_COHORT.GET_COHORT_PENDING });
```

Comment:
Also, as discussed, please create separate action creators for the 3 different actions which are being dispatched in this function for easier testing :)